{{cookiecutter.repostory_name}}/README.md

```diff
@@ -62,10 +62,21 @@ $ ./letsencrypt_setup.sh
 ```
 {% endif %}
 
-Setting up backups
+Setting up periodic backups
 ------------------
 
-Add `cd {{ cookiecutter.repostory_name }}; {{ cookiecutter.repostory_name }}/bin/backup-to-email.sh target@email.address` to crontab. This assumes you have configured email access in `.env` properly.
+Add `cd {{ cookiecutter.repostory_name }}; {{ cookiecutter.repostory_name }}/bin/backup-db.sh` to crontab. 
```

Comment:
Can we use https://blog.sentry.io/2017/11/28/sentry-bash to monitor the backup runners?

Reply:
Do we have sentry-cli installed by default on the app servers? If not, which way to install (from https://docs.sentry.io/product/cli/installation/ ) would we prefer?

Reply:
@ppolewicz @mlech-reef I need guidance here

Reply:
For now, we have `bin/prepare-os.sh` for Ubuntu 20.04. You can extend it to install `sentry-bash` in `/usr/local/bin/` via `wget`/`curl` as this script is supposed to be run with `sudo`.

But when you implement monitoring, it should behave similar to the django monitoring, so only if `SENTRY_DSN` is defined and not empty.

Reply:
I have never used `sentry-bash` so dunno the details of how it works.

Reply:
Added